app/routes/press-releases/press-release.js

```diff
@@ -1,9 +1,14 @@
 import Route from '@ember/routing/route';
+import PressReleaseSnapshot from '../../utils/press-release-snapshot';
 
 export default class PressReleasesPressReleaseRoute extends Route {
-  model(params) {
-    return this.store.findRecord('press-release', params.press_release_id, {
-      include: 'publication-event'
-    });
+  async model(params) {
+    const pressRelease = await this.store.findRecord('press-release', params.press_release_id);
```

Comment:
Het publication-event kan je best al includen zoals voorheen. Dat is sowieso nodig om te bepalen of het persbericht al gepubliceerd is.